### ISSUE TITLE
[JENKINS-61415] Polling baselines cannot be easily removed from Pipelines once added

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/workflow-job-plugin#696]
<p>Once a Pipeline has a polling baseline added to it for a given SCM (shared library, main repo, etc), that baseline cannot be removed directly, and the job will always be built when the SCM is updated.</p>

<p>The only way to remove the baseline is to disable polling for the SCM, build the job, and then restart Jenkins after the build completes. Once that is done, the polling baseline will no longer be present for that SCM and the job will not be built when the SCM is updated.</p>

<p>It should be possible to remove the polling baseline without needing to restart Jenkins.</p>

<p>There are probably some related issues that should be closed as a duplicate of this issue.</p>

---
<details><summary><i>Originally reported by <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/dnusbaum.png" title="dnusbaum's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=dnusbaum">dnusbaum</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-61415" target="_blank">Polling baselines cannot be easily removed from Pipelines once added</a></i></summary>
<i><ul>
<li><b>assignee</b>: <img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/ataylor.png" title="ataylor's avatar" /> <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=ataylor">ataylor</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>component(s)</b>: workflow-job-plugin
<li><b>label(s)</b>: pipeline
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2020-03-10T21:26:07+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 1
<li><b>imported</b>: 20251212-090250
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>Once a Pipeline has a polling baseline added to it for a given SCM (shared library, main repo, etc), that baseline cannot be removed directly, and the job will always be built when the SCM is updated.</p>

<p>The only way to remove the baseline is to disable polling for the SCM, build the job, and then restart Jenkins after the build completes. Once that is done, the polling baseline will no longer be present for that SCM and the job will not be built when the SCM is updated.</p>

<p>It should be possible to remove the polling baseline without needing to restart Jenkins.</p>

<p>There are probably some related issues that should be closed as a duplicate of this issue.</p></pre>
</details>
</details>

**Comment by jenkins-infra-bot** [comment]
- _Remote link associated with this issue:_


  - [jenkinsci/workflow-job-plugin#​152](https://github.com/jenkinsci/workflow-job-plugin/pull/152)

**Comment by jenkins-infra-bot** [comment]
<details><summary><i><img align="left" width="20" src="https://raw.githubusercontent.com/jenkinsci/artifacts-from-jira-issues/refs/heads/main/avatars/dnusbaum.png" title="dnusbaum's avatar" /> dnusbaum:</i></summary>

<ul>
<li><i>Original <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-61415?focusedId=387016&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-387016">comment link</a></i>
<li><details><summary><i>Raw content of original comment:</i></summary>

<pre>
<p>A fix for this issue was released in Pipeline: Job Plugin version 2.37. See the <a href="https://github.com/jenkinsci/workflow-job-plugin/blob/master/CHANGELOG.md#237" class="external-link" target="_blank" rel="nofollow noopener">changelog</a> for details.</p>
</pre>
</details>

</ul>
</details>

<p>A fix for this issue was released in Pipeline: Job Plugin version 2.37. See the <a href="https://github.com/jenkinsci/workflow-job-plugin/blob/master/CHANGELOG.md#237" class="external-link" target="_blank" rel="nofollow noopener">changelog</a> for details.</p>
